spec/factories/rollout_key_value_stores.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :rollout_key_value_store do
+    key "key"
+    value "value"
```

Comment:
Would be worth making these a bit more realistic?

I'm assuming that `key` is the feature name `annotations` and value is some sort of flag (`true`?). I expect there's a good reason, but why isn't value a boolean?